File: push_sale_mobile-master/push_sale_mobile-master/lib/models/actor_profile.dart
class ActorProfile {
  final int id;
  final String code;
  final String name;
  final String name_ar;
  final bool has_stock_mobile;
  ActorProfile({
    required this.id,
    required this.code,
    required this.name,
    required this.name_ar,
    required this.has_stock_mobile,
  });

  static ActorProfile fromMap(Map<String, dynamic> value) {
    return ActorProfile(
      id: value["id"],
      code: value["code"],
      name: value["name"],
      name_ar: value["name_ar"],
      has_stock_mobile: value["has_stock_mobile"] == 1,
    );
  }

  String getName(String lang) {
    switch (lang) {
      case "ar":
        return name_ar;
      default:
        return name;
    }
  }
}

File: push_sale-master/app/Models/ActorProfile.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Laravel\Passport\HasApiTokens;

class ActorProfile extends Model
{
    use HasApiTokens,HasFactory;
    protected $table="actor_profile";

    protected $fillable = ["id","code","name","name_ar","workspace_type","has_stock_mobile","add_client"];

    protected $hidden = ["created_at","updated_at"];
}
